odt2txt/0001-Compile-for-Msys2.patch

```diff
@@ -0,0 +1,28 @@
+From c8d2dced8ce29c2bfcbc07bd302b71b43d3eafb9 Mon Sep 17 00:00:00 2001
+From: "U-SERESCO\\afanjul" <afanjul@AFANJULPORT.seresco.red>
+Date: Sun, 6 Aug 2017 17:56:42 +0200
+Subject: [PATCH] Compile for Msys2
+
+---
+ Makefile | 5 +++++
+ 1 file changed, 5 insertions(+)
+
+diff --git a/Makefile b/Makefile
+index e4d31d9..03e9305 100644
+--- a/Makefile
++++ b/Makefile
+@@ -73,6 +73,11 @@ ifeq ($(UNAME_O),Cygwin)
+ 	LIBS += -liconv
+ 	EXT = .exe
+ endif
++ifeq ($(UNAME_O),Msys)
++	CFLAGS += -I/mingw$(ARCH)/include -I/mingw$(ARCH)/lib/libzip/include
++	LIBS += -liconv -llibzip -L/mingw$(ARCH)/lib -lzip -lz -L/mingw$(ARCH)/lib -L/mingw$(ARCH)/bin
++	EXT = .exe
++endif
+ ifneq ($(MINGW32),)
```

Comment:
This, existing, clause would suggest that you *can* build odt2txt as a MINGW package, i.e. as a pure Win32 application without leaning on POSIX semantics emulated by the MSYS2 runtime. Let's build a MINGW package, then?